example/example.jsx

```diff
@@ -16,6 +16,8 @@ ArsArsenal.render(app, {
 
   onChange(value) {
     console.log("Value changed to %s", value);
-  }
+  },
+
+  multiselect: true
 
 })
```

Comment:
What about making two arsenals? One with single select, and another for multiple?